graylog2-server/src/test/java/org/graylog2/plugin/MessageTest.java

```diff
@@ -392,6 +392,26 @@ public void testValidKeys() throws Exception {
         assertFalse(Message.validKey(" "));
     }
 
+    @Test
+    public void testCleanKey() throws Exception {
+        // Valid keys
+        assertEquals("foo123",Message.cleanKey("foo123"));
```

Comment:
nit: spacing between parameters. There are some other places in this file that could use this same thing so if you could run a Code > Reformat Code it should clean them all up.

Reply:
Thanks for catching this! I went ahead and formatted both test files, so there are some more formatting cleanup included in the commit.